spec.html

```diff
@@ -21777,6 +21777,14 @@
         </emu-note>
       </emu-clause>
 
+      <emu-clause id="sec-hostensurecancompilestrings" aoid="HostEnsureCanCompileStrings">
+        <h1>HostEnsureCanCompileStrings( _callerRealm_, _calleeRealm_ )</h1>
+
+        <p>HostEnsureCanCompileStrings is an implementation-defined abstract operation that allows host environments to block certain ECMAScript functions which allow developers to compile strings into ECMAScript code.</p>
```

Comment:
@michaelficarra @allenwb Were you saying that sections and definitions should be able to start with non-normative paragraphs which are not in notes, Allen? How does that apply here?